### PR TITLE
Update items stored on cryo storage

### PR DESCRIPTION
**Pull request by PIerreLouisH**
## What Does This PR Do
Add every forgot high risk items on the list of items stored on cryo storage.
Also add up items such as stamps.

## Why It's Good For The Game
For once as a Ntr i don't need to ahelp the admins to get a new stamp back to be allowed to fax them in the first place.

## Testing
Spawned went into storage with my stamp, stamp was on the list, all gucci.

## Changelog
:cl:
tweak: Stamps, knuckleduster and captain rapier can be recovered from cryo storage
del: Delete RCD and RPD from cryo storage
/:cl:

### CODE REVIEW

File: code/game/machinery/cryopod.dm [hunk 1]
```diff
@@ -228,9 +228,9 @@
 		/obj/item/autopsy_scanner,
 		/obj/item/holosign_creator/atmos,
 		/obj/item/clothing/gloves/color/black/forensics,
-		/obj/item/rcd,
```
Comment (Henri215) [inline code/game/machinery/cryopod.dm:231]: Add this to the changelog too

**Review by Contrabang — APPROVED** [review]
code is fine

**Review by Henri215 — APPROVED** [review]

**Review by DGamerL — COMMENTED** [review]
Could you add the engraved knuckledusters aswell? I think they currently get deleted

**Comment by Fordoxia** [comment]
> Could you add the engraved knuckledusters aswell? I think they currently get deleted

Oh, also the captain's rapier + sheath isn't preserved either.
Or the plutonium core/container.

File: code/game/machinery/cryopod.dm [hunk 2]
```diff
@@ -228,9 +228,9 @@
...
 		/obj/item/rpd,
```
[the remark this answers is not in the document]
Comment (PIerreLouisH) [reply code/game/machinery/cryopod.dm]: Done

**Review by Contrabang — APPROVED** [review]
lgtm

**Review by DGamerL — APPROVED** [review]

File: code/game/machinery/cryopod.dm [hunk 3]
```diff
@@ -228,9 +228,14 @@
 		/obj/item/autopsy_scanner,
 		/obj/item/holosign_creator/atmos,
 		/obj/item/clothing/gloves/color/black/forensics,
-		/obj/item/rcd,
-		/obj/item/rpd,
-		/obj/item/mod/control
+		/obj/item/mod/control,
+		/obj/item/stamp,
+		/obj/item/melee/knuckleduster,
```
Comment (1080pCat) [inline code/game/machinery/cryopod.dm:233]: Wouldn't this be /all/ dusters? not just the gold one?

Comment (Fordoxia) [inline code/game/machinery/cryopod.dm:233]:
```suggestion
		/obj/item/melee/knuckleduster/nanotrasen,
```

Comment (PIerreLouisH) [reply code/game/machinery/cryopod.dm]: Yes actually, fixed

**Review by Contrabang — APPROVED** [review]

**Review by lewcc — APPROVED** [review]

**Pull request merged**